src/concrete/ml/deployment/fhe_client_server.py

```diff
@@ -25,6 +26,27 @@
     from importlib_metadata import version
 
 
+class Mode(Enum):
+    """Mode for the FHE API."""
+
+    INFERENCE = "inference"
+    TRAINING = "training"
+
+
+def ensure_tuple(x: Union[Any, Tuple[Any, ...]]) -> Tuple[Any, ...]:
```

Comment:
we already have a `to_tuple` function in `concrete.ml.common.utils` that does exactly this !